docs/start/getting_started.rst

```diff
@@ -184,6 +184,95 @@ Now you should be able to access the server at ``http://127.0.0.1:8000/``.
     (kolibri)$ npm rebuild node-sass
 
 
+
+Development and test servers using docker
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+For developers and testers who need to spin up a Kolibri instance quickly without
+setting up the full JavaScript and Python stack, we provide docker images that
+setup all the necessary details and start, either kolibri of the working directory,
+of if supplied with the ENV variable ``KOLIBRI_PEX_URL`` run the specified pex file.
```

Comment:
Just wanna challenge this statement: Would a developer want to spin up a Docker to do development? Under which circumstances is this a good idea? Can the text perhaps state that?

Reply:
I don't think Kolibri developers will be using the dockerfiles for dev work, but might be useful for testing. Running a branch or a PR in an environment completely isolated from the local 
 `node_modules ` and `venv` in the current working directly seems like a useful thing to have.

Use case: checkout PR, run it, test a few things interactively, run tests suite, then stop container and delete it. The `demoserver` approach would be good for this, but running a real dev setup with debug toolbar and logging etc is better.